components/tools/OmeroPy/test/unit/clitest/test_sessions.py

```diff
@@ -50,8 +50,12 @@ def testDefaultSessionsDir(self):
         assert store.dir == path(get_user_dir()) / 'omero' / 'sessions'
 
     @pytest.mark.parametrize('environment', (
-        [], ["OMERO_SESSIONS_DIR"], ["OMERO_SESSIONDIR"],
-        ["OMERO_SESSIONS_DIR", "OMERO_SESSIONDIR"]))
+        [],
+        ["OMERO_SESSIONS_DIR"], ["OMERO_SESSIONDIR"], ["OMERO_USERDIR"],
+        ["OMERO_SESSIONS_DIR", "OMERO_SESSIONDIR"],
+        ["OMERO_SESSIONS_DIR", "OMERO_USERDIR"],
+        ["OMERO_SESSIONDIR", "OMERO_USERDIR"],
+        ["OMERO_SESSIONS_DIR", "OMERO_SESSIONDIR", "OMERO_USERDIR"]))
```

Comment:
What are you testing with `OMERO_SESSIONS_DIR`? It doesn't seem to be in the main code.